.circleci/config.yml

```diff
@@ -55,7 +55,19 @@ references:
     branches:
       ignore: /.*/
 
-version: 2
+orbs:
+  slack: circleci/slack@3.4.2
+
+version: 2.1
+
+commands:
+  send_nightly_build_failed_notification:
+    steps:
+      - slack/status:
+          fail_only: true
+          failure_message: Nightly Build failed. Please check the cause to avoid further issues.
+          include_job_number_field: false
+          webhook: ${OPSCOPS_SLACK_WEBHOOK}
```

Comment:
Need to change this to the team channel's webhook.

Reply:
This `OPSCOPS_SLACK_WEBHOOK` is [teams/next/n-health/continuous-integration in vault](https://vault.in.ft.com:8080/ui/vault/secrets/secret/show/teams/next/n-health/continuous-integration)

Reply:
The team can set `SLACK_WEBHOOK` directly in the project setting in circle